src/SDKs/IotCentral/Management.IotCentral/generate.ps1

```diff
@@ -0,0 +1 @@
+powershell.exe -ExecutionPolicy Bypass -NoLogo -NonInteractive -NoProfile -File "$(split-path $SCRIPT:MyInvocation.MyCommand.Path -parent)\..\..\..\..\tools\generateTool.ps1" -ResourceProvider "iotcentral/resource-manager" -PowershellInvoker  -AutoRestVersion "latest"
```

Comment:
generate.ps1 uses powershell cmdlet now, please take a look at [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/generate.ps1) and make appropriate changes and regenerate the code.

Reply:
I've updated the ps1 script and generated the files again to make sure everything is in sync.